styles/blog.module.scss

```diff
@@ -0,0 +1,18 @@
+.post {
```

Comment:
Remove this file and any unused styles or files.

Reply:
removed